pkgs/development/tools/database/webdis/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, hiredis, http-parser, jansson, libevent }:
+
+stdenv.mkDerivation rec {
+  pname = "webdis";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "nicolasff";
+    repo = pname;
+    rev = version;
+    sha256 = "1hwdmasdr09zlm2pw91wygc87xq2gyg8mi0lb8yqm5nimw5vnzpq";
+  };
+
+  buildInputs = [ hiredis http-parser jansson libevent ];
+
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
```

Comment:
```suggestion
  # Otherwise, the build fails with:
     # mkdir -p //usr/local
     # mkdir: cannot create directory '//usr': Permission denied
     # make: *** [Makefile:45: //usr/local] Error 1
  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
```